### DISCUSSION TITLE
Does the Gateway resource support the setting of socket option?

### DISCUSSION THREAD

**Discussion by FamineSeekers** [kubernetes-sigs/gateway-api#2010] [Q&A]
The IP address, port number, and socket option of the listener can be set in the static configuration of the Envoy. Can the dynamic configuration be supported?

**Comment by howardjohn** [comment]
Port number is part of listener. The rest is sort of under spec.address, but I suspect not in the way you want it for most implementations (would be the Service IP, not the IP the pod binds to?)
